.github/workflows/build-and-test-toolchain.yml

```diff
@@ -155,6 +160,13 @@ jobs:
         run: |
           cat ${{ steps.get-wsl-paths.outputs.artifact-path }}\gcc-tests-${{ env.TAG }}\summary.txt >> $env:GITHUB_STEP_SUMMARY
 
+      - name: Group test failures
+        run: |
+          cd ~/work
+          source .github/scripts/config.sh
+          python .github/scripts/toolchain/group-gcc-test-failures.py --dir $ARTIFACT_PATH/gcc-tests-${{ env.TAG }} >> \
```

Comment:
I would move it to a separate script.

Reply:
What do you mean by separate script? Like to move this action step into separate `.yml` file? Or to create "top 10" from https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/pull/184#discussion_r1729086014 in separate `.py` script?

Reply:
create a script. something like toolchain/group-gcc-test-failures.sh

Reply:
@vejbomar Note that `cd ~/work` command should stay here as it is `Vampire/setup-wsl@v3` action environment-specific workaround and do not belong to any script that can be used idependently.

Reply:
what is it needed? what will be starting folder without using `cd ~`?

Reply:
Root folder of the repository. This workflow cannot checkout the repository sources directly to \`$GITHUB\_WORKSPACE\` so they are checked out to \`~/work\` folder and the \`cd ~/work\` is needed.

Reply:
Then a separate script could be called directly with one line
~/work/.github/scripts/toolchain/group-gcc-test-failures.sh

Reply:
> Then a separate script could be called directly with one line ~/work/.github/scripts/toolchain/group-gcc-test-failures.sh

Yes, but it would have to be assured that the `group-gcc-test-failures.sh` and the `config.sh` uses only absoute paths.